src/blockMonitor/reorgDetector.ts

```diff
@@ -131,18 +142,17 @@ export class ReorgDetector extends StartStopService {
         // and emit the end reorg event
         this.emit(ReorgDetector.REORG_END_EVENT, newHead.height);
         this.provider.polling = true;
+
+        this.conductingReorg = false;
     }
 
     /**
      * Since this reorg detector only detects reorgs below a max depth it can prune records
      * that it has below that
      */
     private prune() {
-        // prune the chain
-        const minHeight = this.headBlock.height - this.maxDepth;
-        this.headBlock.prune(minHeight);
```

Comment:
What happened to this line?

Reply:
I'm not sure why this is here, but those lines are still there in the current version of the code (in reorgDetector.ts instead of reorg.ts).

Reply:
The following line appears to be missing in reorgDetector.ts.
```
this.headBlock.prune(minHeight);
```

Reply:
It's only pruning the store for the height events:
```
    /**
     * Since this reorg detector only detects reorgs below a max depth it can prune records
     * that it has below that
     */
    private prune() {
        // prune current re-org height listeners
        const minHeight = this.headBlock.height - this.maxDepth;
        this.store.prune(minHeight);
    }
```